src/node_contextify.cc

```diff
@@ -107,6 +107,17 @@ ContextifyContext::ContextifyContext(
 }
 
 
+bool ContextifyContext::AllowWasmCodeGeneration(
+    Local<Context> context, Local<String>) {
+  Local<Value> wcg = context->GetEmbedderData(kAllowWasmCodeGeneration);
+  if (!wcg->IsUndefined()) {
+    CHECK(wcg->IsBoolean());
+    return wcg.As<Boolean>()->Value();
+  }
+  return true;
```

Comment:
Just `return wcg->IsUndefined() || wcg->IsTrue();`?

Aside: `wasm_code_generation`, or `wasm_code_gen` if you think that's too verbose, but no abbrevs.